src/basic/options.fs

```diff
@@ -187,7 +187,15 @@ let get_include_path (dirname:string) =
     !_include_path@["."; h ^ "/lib"; h ^ "/lib/fstar"])
 
 let prims () = match !prims_ref with
-  | None -> Util.format1 "%s/lib/prims.fst" (get_fstar_home ())
+  | None -> 
+    (let filen = "prims.fst" in
```

Comment:
I think these parentheses are actually necessary. I tried removing them and F# gets confused because of the nested match statements.